src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rent = 40;
+  const sale1 = 20;
+  const sale2 = 50;
+  const duration1 = 3;
+  const duration2 = 7;
+
+  if (days < duration1) {
+    return days * rent;
+  }
```

Comment:
```suggestion
  if (days < duration1) {
    return (days * rent);
  }
```
use expressions in brackets